pageserve/templates/base.html

```diff
@@ -32,44 +34,60 @@
 </head>
 
 <body>
+    <div class="topnav">
+        <div id="nav_left">
+            <a class="{% block home_tab_active %}{% endblock %} navtab navbtn" href='/v1/'>Home</a>
+            <a class="{% block events_tab_active %}{% endblock %} navtab navbtn" href='/v1/events'>Events</a>
+        </div>
+        <div id="user_nav">
+            {% block login_buttons %}
+            {% if not session['user'] %}
+            <div id="sign_in_button">
+                <div class="g-signin2" data-onsuccess="onSignIn" data-theme="dark"></div>
+                <script>
+                    function onSignIn(googleUser) {
+                        // Secure Google ID token to send to backend for verification.
+                        var id_token = googleUser.getAuthResponse().id_token;
+                        var xhr = new XMLHttpRequest();
+                        xhr.open('POST', "authenticate");
+                        xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+                        xhr.onload = function () {
+                            // show the "Sign out" button
+                            window.location.reload()
+                        };
+                        xhr.send('gauth_token=' + id_token);
+                    }
+                </script>
+            </div>
+            {% else %}
+            <div class='navtab'>Hi {{ session['user']['name'] }}!</div>
+            <a href="#" onclick="signOut();" id="sign_out_button" class='navtab navbtn'>Sign out</a>
+            <script>
+                function signOut() {
+                    var auth2 = gapi.auth2.getAuthInstance();
+                    auth2.signOut().then(function () {
+                        window.location.replace("/v1/sign_out");
+                    });
+                }
+            </script>
+            {% endif %}
+            {% endblock login_buttons %}
+        </div>
+    </div>
+
+    <div style="padding: 0px 15px">
     <header>
         {% block header %}{% endblock header %}
     </header>
-    {% block login_buttons %}
-    {% if not session['user'] %}
-    <div id="sign_in_button">
-        <div class="g-signin2" data-onsuccess="onSignIn" data-theme="dark"></div>
-        <script>
-            function onSignIn(googleUser) {
```

Comment:
It looks like something (probably the extra `div` tags) confused the github diff.

Not a big deal, but feel free to ignore comments above on re-indented but otherwise unchanged code.